client/components/public/imports/HomeEarlyBird.jsx

```diff
@@ -44,26 +44,33 @@ class HomeEarlyBird extends Component {
             </Grid.Column>
            </Grid.Row>
            <Grid.Row centered>
+           <p>Registration opens {regularRegistrationStart}, and ends {regularRegistrationEnd}.</p>
+           </Grid.Row>
+           <Grid.Row centered>
            {registerButton}
            <br/>
            </Grid.Row>
             <Grid.Row centered width={16} columns={2}>
-              <Grid.Column>
+            <p>Don't have a team? See our <a href="/faq">FAQ</a></p>
+              {/* <Grid.Column>
                  <Header size="medium" style={{color: "white"}}>Early Bird Registration</Header>
                   {registrationOpenDate} through {earlyBirdLastDate}<br /><br />
                   Student   $5<br/><br/>
                   Non-Student   $10<br/><br/><br/>
-               </Grid.Column>
+               </Grid.Column> */}
 
-                <Grid.Column>
+                {/* <Grid.Column>
                   <Header size="medium" style={{color: "white"}}>Regular Registration</Header>
                     {regularRegistrationStart} through {regularRegistrationEnd}<br/><br/>
                     Student   $8<br/><br/>
                     Non-Student   $15<br/><br/>
-                </Grid.Column>
+                </Grid.Column> */}
+                
             </Grid.Row>
-            <Grid.Row centered style={{fontSize: "12pt"}}>
-              *Registration prices are per person. Each person wanting to join a team must register themselves first.<br /><br />
+            <Grid.Row style={{fontSize: "12pt"}}>
+                {/* *Registration prices are per person. Each person wanting to join a team must register themselves first.<br /><br /> */}
```

Comment:
we can prob delete this line instead of commenting out

Reply:
Fixed!